addons/website_slides/controllers/main.py

```diff
@@ -1331,9 +1464,19 @@ def _get_channels(self, **kwargs):
             channels = tools.lazy(lambda: request.env['slide.channel'].browse(int(kwargs['channel_id'])))
         return channels
 
+    def _get_invite_error_msg(self, **kwargs):
```

Comment:
No need to have more than one of these.
```suggestion
    @staticmethod
    def _get_invite_error_msg(**kwargs):
```